services/notification/src/app.py

```diff
@@ -254,7 +254,31 @@ def null_to_none(s):
 
 
 def string_to_bool(s):
-    return None if s == 'null' else s == 'true'
```

Comment:
This was the source of the random bug, the test fixtures sometimes sent "True" vs. "true"